app/pages/lab/subject-set.cjsx

```diff
@@ -154,8 +156,8 @@ EditSubjectSetPage = React.createClass
           <strong>Manifest files are required.</strong><br />
           <strong>Drag-and-drop or click to upload manifests and subject images here.</strong><br />
           Manifests must be <code>.csv</code> or <code>.tsv</code>. The first row should define metadata headers. All other rows should include at least one reference to an image filename in the same directory as the manifest.<br />
-          Subject images can be any of: {<span key={ext}><code>{ext}</code>{', ' if VALID_SUBJECT_EXTENSIONS[i + 1]?}</span> for ext, i in VALID_SUBJECT_EXTENSIONS}{' '}
-          and may not contain {<span key={char}><kbd>{char}</kbd>{', ' if INVALID_FILENAME_CHARS[i + 1]?}</span> for char, i in INVALID_FILENAME_CHARS}.<br />
+          Subject images can be {MAX_FILE_SIZE/1000}kb and any of: {<span key={ext}><code>{ext}</code>{', ' if VALID_SUBJECT_EXTENSIONS[i + 1]?}</span> for ext, i in VALID_SUBJECT_EXTENSIONS}{' '} 
```

Comment:
Pedantic: lowercase "kb" is kilobits. Can we make this "KB"? And maybe "can be up to _n_ KB" is clearer?